src/main/resources/pagedump/galen-pagedump.js

```diff
@@ -207,6 +207,24 @@ Suggest.prototype.generateSuggestions = function (items) {
     return "<pre>" + html + "</pre>";
 };
 Suggest.prototype._suggestions = {};
+Suggest.prototype._suggestions.width = function(a, b) {
+           return new Spec(a.name, "width: " + getPercentage(a.area[2], b.area[2]) + "% of " + b.name+"/width");
+   };
+Suggest.prototype._suggestions.height = function(a, b) {
+          return new Spec(a.name, "height: " + getPercentage(a.area[3], b.area[3]) + "% of " + b.name+"/height");
+   };
+Suggest.prototype._suggestions.widthAgainstHeight = function(a, b) {
+    return new Spec(a.name, "width: " + getPercentage(a.area[2], b.area[3]) + "% of " + b.name+"/height");
```

Comment:
Why would you ever need to check the ration of one object width to a height of another object? I have never heard of such situation and personally never had the need to testing this.